packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -67,6 +67,7 @@ export const ChatList = ({
   const [chatListClient, setChatListClient] = useState<StatefulGraphChatListClient | undefined>();
   const [chatListState, setChatListState] = useState<GraphChatListClient | undefined>();
   const [menuItems, setMenuItems] = useState<ChatListMenuItem[]>(props.menuItems === undefined ? [] : props.menuItems);
+  const [isSignedIn, setIsSignedIn] = useState<boolean>(false);
```

Comment:
Are we sure we don't have a re-render on sign-in anyway? I thought that changed the state of the chatListClient or chatListState anyway?

Reply:
Not sure where you are getting at, but here we are just setting the initial value, which is following the pattern for all other state entities, no?